bloom-housing/bloom
# 1728/notifications qa

# Pull Request Template

## Issue

Addresses #1728


## Description

QA Notes from Notifications - includes changing the icons on the homepage and giving the action block section a max width.

## Type of change


## How Can This Be Tested/Reviewed?

Check out the homepage with Alameda as your jurisdiction and with a different jurisdiction to see both states

## Checklist:
